src/Symfony/Component/Console/Question/Question.php

```diff
@@ -130,14 +130,11 @@ public function getAutocompleterValues()
     /**
      * Sets values for the autocompleter.
      *
-     * @param iterable|null $values
-     *
      * @return $this
      *
-     * @throws InvalidArgumentException
```

Comment:
lower

Reply:
The code that throws the exception is removed by this PR. Lower branches still have it.